deploy/rp-production.json

```diff
@@ -242,6 +242,59 @@
                 "[resourceId('Microsoft.Network/loadBalancers', 'rp-lb')]"
             ]
         },
+        {
+            "properties": {
+                "groupShortName": "rphealth",
+                "enabled": true
+            },
+            "name": "rphealth-ag",
+            "type": "Microsoft.Insights/ActionGroups",
+            "location": "Global",
+            "apiVersion": "2018-03-01"
+        },
+        {
+            "properties": {
+                "severity": 4,
+                "enabled": true,
+                "scopes": [
+                    "[resourceId('Microsoft.Network/loadBalancers', 'rp-lb')]"
+                ],
+                "evaluationFrequency": "PT5M",
+                "windowSize": "PT15M",
+                "targetResourceType": "Microsoft.Network/loadBalancers",
+                "criteria": {
+                    "allOf": [
+                        {
+                            "operator": "LessThan",
+                            "threshold": 99,
```

Comment:
I think our SLA is 3 9's, so as Jim pointed out, we might want to set a more ambitious target for SLOs (99.95?)

What time interval is this calculated on? I see "average" here but not the window for the average.

Reply:
"windowSize": "PT15M", - 15min

Reply:
Changed to 99.95 and 1h window for now.
Allowed values are: "PT1M","PT5M","PT15M","PT30M","PT1H","PT6H","PT12H","PT24H"

Reply:
That's just 3s of downtime for a 1h window... can we make it 95% per 1H which would be 3m of downtime?

Reply:
Set at 95% of 1h.